lib/board.ex

```diff
@@ -1,7 +1,5 @@
 defmodule Board do
-  @moduledoc """
-    This is the game's visual engine.
-  """
+  @moduledoc false
```

Comment:
Do you need `@moduledoc false` here? Can't you just set a flag on credo telling it to ignore this rule?